bin/alluxio-start.sh

```diff
@@ -26,7 +26,6 @@ Where ACTION is one of:
   masters                   \tStart masters on master nodes.
   proxy                     \tStart the proxy on this node.
   proxies                   \tStart proxies on master and worker nodes.
-  safe                      \tScript will run continuously and start the master if it's not running.
```

Comment:
I do see other code related, will check that part, but I think it can be done in a following pr.
Need to double check with others, let's do that in the meeting.